lib/customer.rb

```diff
@@ -0,0 +1,43 @@
+class Customer
+  attr_reader :name, :address, :phone, :pizza_order
+
+  def initialize(name, address, phone)
+    @name = name
+    @address = address
+    @phone = phone
+    @order_frequency = 0
+    @order = {}
+    @order_time = nil
+    @pizza_order = nil
+  end
```

Comment:
Think about which of these are truly attributes of the class and which are just calculated values.